pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -872,7 +873,8 @@ func (c *Reconciler) createRunObjects(ctx context.Context, rpt *resources.Resolv
 	var runObjects []v1beta1.RunObject
 	ctx, span := c.tracerProvider.Tracer(TracerName).Start(ctx, "createRunObjects")
 	defer span.End()
-	matrixCombinations := rpt.PipelineTask.Matrix.FanOut()
+	rpt.PipelineTask.Matrix.FanOut()
```

Comment:
it seems you meant to delete this line, the function is actually used in 877 below
```suggestion

```